test/feature/dynamic_types/DynamicTypesTests.cpp

```diff
@@ -12922,7 +12922,7 @@ TEST_F(DynamicTypesTests, DynamicType_XML_struct_with_enum)
     ASSERT_EQ(RETCODE_OK,
             DomainParticipantFactory::get_instance()->load_XML_profiles_file(DynamicTypesTests::config_file()));
 
-    DynamicType::_ref_type xml_type;
+    DynamicTypeBuilder::_ref_type xml_type;
```

Comment:
NIT, adjust variable names to be coherent 
```suggestion
    DynamicTypeBuilder::_ref_type xml_type_builder;
```